FIAT/barycentric_interpolation.py

```diff
@@ -0,0 +1,35 @@
+# This is work in progress
```

Comment:
Is this still the case?

Can you add the SPDX license header?

Reply:
I'm saying it's still a WIP because I'm not even sure if what I do for higher order derivatives (multiplying D*...*D) is a sensible thing to do. Compared to the three term recurrence it's already good for the first derivate, but I should run my example with both approaches for the norms H^1, H^2, H^3, etc.

Reply:
Should we add a test for this?